docs/development.md

```diff
@@ -64,58 +51,57 @@ $ pnpm install
 
 ## Building
 
-You can now finally build the Docker images using the following command:
-
-```sh
-$ pnpm dev build
-```
-
-_This might take a surprisingly long time_. Thankfully, Docker's build step is _heavily_ cached. It won't take this long again (unless of course you need to change one of those early steps).
+You can now finally build the Docker images. First, start docker:
 
-## Development
+<table>
+<thead><tr><th>Systemd Distros</th><th>WSL2</th></tr></thead>
+<tbody valign="top">
+<td><p><pre>$ sudo systemctl --user start docker-desktop</pre></p></td>
```

Comment:
Oh I see the confusion now. Hmm, not sure about `docker-desktop` specifically? I think it would be better, in setup, to suggest turning on Docker as a _daemon_ (i.e. `sudo systemctl enable --now docker.service`) and then also mentioning that Docker desktop is a thing later.

"Start" means run now (but on restart it'll be gone).
"Enable" means automatically run on startup.
"Enable now" means automatically run on startup, and if it's not running right now, also start it.